import math
def calculate_error_function(x):
        return math.erf(x)
n = 10
print("Powers of 2:", [2**x for x in range(n)])